src/validation.cpp

```diff
@@ -80,6 +80,8 @@ bool CBlockIndexWorkComparator::operator()(const CBlockIndex *pa, const CBlockIn
 
 CChainState g_chainstate;
 
+CChainState& ChainstateActive() { return g_chainstate; }
```

Comment:
in commit 4d66886 refactoring: introduce ChainstateActive()

How about making `g_chainstate` static?